bogota-abril-2019/css-magazine-cover/Álvaro David Pongutá/index.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html lang="en" class="background-image">
```

Comment:
Avoid the used class on the html tag, if the case used it in the body.

Reply:
Corrected